### PR TITLE
WillD enhancements Login and out and upload Avatar

### PR DESCRIPTION
**Pull request by wdillard**
Closes Issue #40

### CODE REVIEW

File: client/src/components/ProfilePage.jsx [hunk 1]
```diff
@@ -141,6 +164,10 @@ const ProfilePage = ({ history: { push } }) => {
           <BsTrash size={15} style={{ color: 'rgba(5, 95, 158, 1)' }} /> Delete
           Account
         </button>
+        <button className="logout" onClick={handleLogout}>
+          <VscSignOut size={15} style={{ color: 'rgba(5, 95, 158, 1)' }} />{' '}
```
Comment (ErnieAtLYD) [inline client/src/components/ProfilePage.jsx:168]: I think I see why you added `{' '}` - so you can give some breathing room between the two buttons, right? Honestly, I would just give them a little bit of left and right margins in CSS instead.

File: client/src/components/SignUpPage.jsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ const SignUpPage = ({ history }) => {
               onChange={handleChange}
             />
           </div>
-          <div>
+          {/* <div>
```
Comment (ErnieAtLYD) [inline client/src/components/SignUpPage.jsx:78]: My pet peeve is students leaving giant blocks of commented out code. All I ask is that when this project is officially done, you remove all of the comments on the code blocks you didn't use.

**Review by ErnieAtLYD — APPROVED** [review]
Honestly, from what I can tell everything looks really good. I am not gonna merge this though because I do have a couple of nitpicks that I'll leave up to you to add commits to this branch.

File: client/src/components/SignUpPage.jsx (hunk 2 above)
Comment (wdillard) [reply client/src/components/SignUpPage.jsx:78]: I agree. This was just for testing. I plan to clean everything up.

File: client/src/components/ProfilePage.jsx (hunk 1 above)
Comment (wdillard) [reply client/src/components/ProfilePage.jsx:168]: Thank you. I will look into it.

**Review by Kaylee2319 — APPROVED** [review]

**Pull request merged**